bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/NativeInterface.cs

```diff
@@ -150,6 +150,71 @@ internal static unsafe class NativeInterface
 
         internal unsafe struct CharPtr { };
 
+        #region Collections
```

Comment:
Is this 2,000+ line file code generated, or is there some other compelling reason not to break it up?  I just consider regions a code smell, but I guess interop is often a bit messy.

Reply:
it is not generated and i was intending to keep all of the mess in one place. i'm good with refactoring this at some point in the future.